### ISSUE TITLE
i dont feel the smoothness of fsr 3 generated frames in dying light 2 and cyberpunk.

### ISSUE THREAD

**Issue by juhannnnnn** [Nukem9/dlssg-to-fsr3#388]
i have 10700k and rtx 3070 with 32 gb ram. i have tried lots of combinations like control panel vsync and gsync on and off but to no avail.

**Comment by SamBh1** [comment]
Same with Cyberpunk 2077. 11400f, 3080 10 GB, 16GB RAM. The FPS counter went from around 60 to 100 but the game is not smoother. I also tried enabling vsync for the game in Nvidia control panel and capped fps to my monitor 165hz with RTSS and still nothing.

**Comment by TimmyP777** [comment]
CP2077 HDR color saturation has to be at zero (still looks fine) or it does not work like it should. Its in "HDR settings" in the game.

**Comment by juhannnnnn** [comment]
> CP2077 HDR color saturation has to be at zero (still looks fine) or it does not work like it should. Its in "HDR settings" in the game.

i have hdr disabled

**Comment by morgan-blondellet** [comment]
no, you keep the latency of your base framerate, so it is normal for you that it does not feel smoother.

**Comment by juhannnnnn** [comment]
> no, you keep the latency of your base framerate, so it is normal for you that it does not feel smoother.

huh? pls explain in more details

**Comment by SamBh1** [comment]
> no, you keep the latency of your base framerate, so it is normal for you that it does not feel smoother.

Now that I compared it on vs off again I do in fact see a difference with it being smoother but it isn't as smooth as real 110fps since that's what I expected. while my fps almost doubled from 60~ to 110~ it feels like I only went up to 75-80 in terms of image smoothens.

TLDR: It is a bit smoother but not as smooth as I expected it to be.

**Comment by TimmyP777** [comment]
> no, you keep the latency of your base framerate, so it is normal for you that it does not feel smoother.

This is misleading because you need to factor in Reflex, which is pretty much eliminating all lag that you might complain about.

AMD Antilag+ does not work properly yet, so in the case of AMD what you say is accurate, but not with this.
